starter-code/public/scripts/article.js

```diff
@@ -164,4 +164,4 @@ var app = app || {};
       .then(callback);
   };
   module.Article = Article;
-})(app);
+}(app));
```

Comment:
It's more common to see parens around the function (`(function (module) { ... })(app)`). IIRC, some browsers don't like the format you have here.